backend/courses/filters.py

```diff
@@ -176,6 +204,7 @@ def filter_queryset(self, request, queryset, view):
             "course_quality": bound_filter("course_quality"),
             "instructor_quality": bound_filter("instructor_quality"),
             "difficulty": bound_filter("difficulty"),
+            "is-open": is_open_filter,
```

Comment:
Might want to make this `is_open` for the sake of consistency

Reply:
Good point